<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FuncDlg</class>
 <widget class="QDialog" name="FuncDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#FuncDlg {
	border-image: url(:/shareImages/dlg_background.png);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="4">
    <widget class="TitleBarWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="inputWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>560</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>560</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>140</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>375</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#widget {
	background-color: rgb(255, 255, 255);
}

QRadioButton::indicator::checked{
border-image:url(:/setImage/radio_select.png);
}
</string>
        </property>
        <widget class="QLabel" name="noLabel">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>17</y>
           <width>71</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font:14px;
font-family:&quot;微软雅黑&quot;;
color: rgb(51, 51, 51);</string>
         </property>
         <property name="text">
          <string>员工工号:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="noValueLabel">
         <property name="geometry">
          <rect>
           <x>92</x>
           <y>17</y>
           <width>151</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font:14px;
font-family:&quot;微软雅黑&quot;;
color: rgb(51, 51, 51);</string>
         </property>
         <property name="text">
          <string>0001</string>
         </property>
        </widget>
        <widget class="QLabel" name="nameLabel">
         <property name="geometry">
          <rect>
           <x>4</x>
           <y>97</y>
           <width>81</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font:14px;
font-family:&quot;微软雅黑&quot;;
color: rgb(51, 51, 51);</string>
         </property>
         <property name="text">
          <string>*员工姓名:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="passwdLabel">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>142</y>
           <width>71</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font:14px;
font-family:&quot;微软雅黑&quot;;
color: rgb(51, 51, 51);</string>
         </property>
         <property name="text">
          <string>*员工密码:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="telLabel">
         <property name="geometry">
          <rect>
           <x>16</x>
           <y>186</y>
           <width>71</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font:14px;
font-family:&quot;微软雅黑&quot;;
color: rgb(51, 51, 51);</string>
         </property>
         <property name="text">
          <string>手机号码:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="powerLabel">
         <property name="geometry">
          <rect>
           <x>6</x>
           <y>222</y>
           <width>81</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font:14px;
font-family:&quot;微软雅黑&quot;;
color: rgb(51, 51, 51);</string>
         </property>
         <property name="text">
          <string>*角色权限:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="passwdEdit">
         <property name="geometry">
          <rect>
           <x>98</x>
           <y>143</y>
           <width>150</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/shareImages/editRect.png);
padding-left:10px;</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="telEdit">
         <property name="geometry">
          <rect>
           <x>99</x>
           <y>187</y>
           <width>150</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/shareImages/editRect.png);
padding-left:10px;</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton1">
         <property name="geometry">
          <rect>
           <x>99</x>
           <y>229</y>
           <width>51</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font:14px;
font-family:&quot;微软雅黑&quot;;
color: rgb(51, 51, 51);

</string>
         </property>
         <property name="text">
          <string>店长</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton2">
         <property name="geometry">
          <rect>
           <x>158</x>
           <y>229</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font:14px;
font-family:&quot;微软雅黑&quot;;
color: rgb(51, 51, 51);</string>
         </property>
         <property name="text">
          <string>收银员</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton3">
         <property name="geometry">
          <rect>
           <x>99</x>
           <y>261</y>
           <width>89</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font:14px;
font-family:&quot;微软雅黑&quot;;
color: rgb(51, 51, 51);</string>
         </property>
         <property name="text">
          <string>导购员</string>
         </property>
        </widget>
        <widget class="QPushButton" name="saveButton">
         <property name="geometry">
          <rect>
           <x>27</x>
           <y>305</y>
           <width>220</width>
           <height>44</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>220</width>
           <height>44</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#saveButton {
	font:14px;
	color: rgb(255, 255, 255);
	border-image: url(:/shareImages/sure_normal.png);
}

QPushButton#saveButton:pressed {
	border-image: url(:/shareImages/sure_press.png);
}</string>
         </property>
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
        <widget class="QLabel" name="tipLabel">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>283</y>
           <width>221</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font:12px;
font-family:&quot;微软雅黑&quot;;
color: #ff0000;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="SearchEdit" name="nameWidget" native="true">
         <property name="geometry">
          <rect>
           <x>97</x>
           <y>98</y>
           <width>150</width>
           <height>30</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget#nameWidget{
	border: 1px solid #EA8852;
}
</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="Line" name="line">
     <property name="minimumSize">
      <size>
       <width>2</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>2</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(102, 102, 102);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QWidget" name="keyboardWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>44</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>193</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="WndDigitalKeyboard" name="keyboardWgt" native="true">
        <property name="minimumSize">
         <size>
          <width>303</width>
          <height>303</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WndDigitalKeyboard</class>
   <extends>QWidget</extends>
   <header location="global">WndDigitalKeyboard.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SearchEdit</class>
   <extends>QWidget</extends>
   <header>SearchEdit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TitleBarWidget</class>
   <extends>QWidget</extends>
   <header>TitleBarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
